modules/common/macro.h

```diff
@@ -15,7 +15,7 @@
  *****************************************************************************/
 
 /**
- * @file
+ * @file macro.h
```

Comment:
you can remove macro.h
doxygen can fill in this field automatically

Reply:
@startcode
thank you. Fixed.